packages/backend/src/accounting/tigerbeetle/service.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line
+const siphash = require('siphash')
```

Comment:
Used require due to the way the module was declared for `siphash`. Suggestions welcome!